client/me/help/help-flow/index.jsx

```diff
@@ -0,0 +1,243 @@
+				<MeSidebarNavigation />
+				<HeaderCake onClick={ this.backToHelp } isCompact>
+					{ translate( 'Contact Us' ) }
+				</HeaderCake>
+				<Card className={ helpFlowClassNames }>
+					<div className="help-flow__wrapper">
+						<img
+							className="help-flow__illustration"
+							alt=""
+							src="/calypso/images/illustrations/contact-us.svg"
+						/>
+						<div className="help-flow__description">
+							<h2 className="help-flow__heading">{ translate( 'Hi there! How can we help?' ) }</h2>
+							<p>
+								{ translate(
+									"Please select the service which you're having issues with, so you can be directed to the ideal place for support."
+								) }
+							</p>
+							<p>
+								{ translate(
+									"If you're unsure of the service you require assistance with, click {{link}}here{{/link}}.",
+									{
+										components: {
+											link: <a href="/help/contact/form" />,
+										},
+									}
+								) }
+							</p>
+						</div>
+					</div>
+					<div className="help-flow__button-wrapper">
+						{ hasWpComSites && (
+							<Button className="help-flow__button is-wpcom" href="/help/contact/form">
+								{ translate( 'My WordPress.com site' ) }
+							</Button>
+						) }
+						{ hasJetpackSites && (
+							<Button
+								className="help-flow__button is-jetpack"
+								primary
+								onClick={ this.handleJetpackButtonClick }
+							>
+								{ translate( 'My Jetpack-connected site' ) }
+							</Button>
+						) }
+						<Button className="help-flow__button is-link" onClick={ this.handleOtherClick }>
+							{ translate( 'Other' ) }
+						</Button>
+					</div>
+					{ hasJetpackSites && hasWpComSites && (
+						<div className="help-flow__site-selector">
+							<p className="help-flow__site-selector-subtitle">
+								{ translate( 'Your Jetpack-connected sites:' ) }
+							</p>
+							<SiteSelector filter={ this.siteFilter } displaySearch={ false } />
+						</div>
+					) }
+				</Card>
+
+				{ displayDialog && (
+					<Dialog isVisible additionalClassNames="help-flow__dialog">
+						<div className="help-flow__dialog-text">
+							<h1 className="help-flow__dialog-heading">
+								{ ! jetpackDialog
+									? translate( 'How can we help?' )
+									: translate( 'We think we could provide more help elsewhere…' ) }
+							</h1>
+							{ dialogContent }
+						</div>
+						<div className="help-flow__dialog-buttons">
+							<Button className="is-link" href="/help/contact/form">
+								{ ! jetpackDialog
+									? translate( "I'd like to discuss something else and speak to a human" )
+									: translate( "I'd like to speak with a human anyway" ) }
+							</Button>
+							<Button className="help-flow__close-dialog" onClick={ this.handleDialogClosure }>
+								{ translate( 'Close' ) }
+							</Button>
+							{ jetpackDialog && (
+								<Button primary onClick={ this.handleDialogClosure }>
+									{ translate( 'Contact Jetpack Support' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **36** times:
`translate( 'Contact Jetpack Support.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).